src/main/resources/inspectionDescriptions/ParameterEqualsDefaultValueInspection.html

```diff
@@ -0,0 +1,6 @@
+<html>
+    <body>
+        <p>Suggests to remove the parameter assignment if it is the same of declared as default.</p>
```

Comment:
Reports arguments which can be safely dropped, as they are identical to the default value.